cluster-provision/gocli/opts/k8sprovision/conf/crio-yum.repo

```diff
@@ -0,0 +1,6 @@
+[isv_kubernetes_addons_cri-o_stable_v1.28]
+name=CRI-O v1.28 (Stable) (rpm)
```

Comment:
I think we will need some version handling here - cri-o version should match the k8s version being deployed.

Reply:
yeah agreed, after inspecting the existing state of the code. some providers share crio version but occasionally one provider might deviate. will account for this